### PR TITLE
[lit-next] Prepare next pre-releases

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Review by kevinpschaaf — CHANGES_REQUESTED** [review]
Need to update the `litHtmlVersions` and friends right?

**Comment by justinfagnani** [comment]
> Need to update the litHtmlVersions and friends right?

done

**Review by sorvell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,18 +11,24 @@ following links for details on the changes being made:
...
+- [`lit-starter-ts`](./packages/lit-starter-ts)
+- [`lit-starter-js`](./packages/lit-starter-js)
 - [`@lit/reactive-element`](./packages/reactive-element)
-- [`lit-ssr`](./packages/lit-ssr)
-- [`tests`](./packages/tests)
-- [`benchmarks`](./packages/benchmarks)
+- [`@lit-labs/ssr`](./packages/lit-ssr)
+- [`@lit-labs/react`](./packages/labs/react)
+- [`@lit-labs/task`](./packages/labs/task)
+- [`localize`](./packages/localize)
```
Comment (kevinpschaaf) [inline README.md:23]: I guess if we're making the link text the npm package name, it should be
```suggestion
- [`@lit/localize`](./packages/localize)
```

**Review by kevinpschaaf — APPROVED** [review]
LGTM, small readme nit

**Pull request merged**